configs/partials/webpack.config.base.js

```diff
@@ -70,22 +61,31 @@ module.exports = {
                 extensions,
             }),
         ],
+        fallback: {
+            crypto: require.resolve('crypto-browserify'),
+            stream: require.resolve('stream-browserify'),
+        },
     },
 
     optimization: {
-        namedModules: true,
-        noEmitOnErrors: false,
+        moduleIds: 'named',
+        emitOnErrors: true,
     },
 
     plugins: [
         new webpack.EnvironmentPlugin({
-            NODE_ENV: 'production',
             TARGET_NET: 'mainnet',
             LEDGER_EMULATOR_URL: '',
+            DEBUG_PROD: false,
+            START_MINIMIZED: false,
+            E2E_BUILD: false,
         }),
         new WasmPackPlugin({
             crateDirectory: path.resolve(__dirname, '.'),
         }),
```

Comment:
Don't know if this is out of the scope of this, but maybe we can move this to only be present in the `renderer.(dev|prod).js`? Right now it builds every time any of the 3 builds happen.

Reply:
I have moved it to each of the renderer configuration files.